src/kata1.fizzBuzz.js

```diff
@@ -1,3 +1,19 @@
-const fizzBuzz = (number) => {};
+const fizzBuzz = (number) => {
+  const multipleOf3 = number % 3 === 0;
+  const multipleOf5 = number % 5 === 0;
+
+  if (multipleOf3 && multipleOf5) {
+    return "FizzBuzz";
+  }
+  if (multipleOf3) {
+    return "Fizz";
+  }
+  if (multipleOf5) {
+    return "Buzz";
+  }
+  if (!(multipleOf3 || multipleOf5)) {
+    return number;
+  }
```

Comment:
Eslint is complaining a bit here, as you do not have a return statement at the root of the function. This is what we call a default exit condition, meaning a condition that will always be hit if no other statements before this have been reached, and its generally a good practice as it means there will be no situation where no value is returned from your function

In this case all we need to do is remove the final `if`  statement and just have it as a return by itself:
```
  return number;
```